plotly/express/_core.py

```diff
@@ -1003,7 +1003,7 @@ def one_group(x):
     return ""
 
 
-def apply_default_cascade(args):
+def apply_default_cascade(args, constructor=None):
```

Comment:
small nit, but I'd prefer not setting a default value for `constructor` here, simply for consistency with the other functions in this file.

`apply_default_cascade(args)` is called in one other place, in `plotly/express/_imshow.py`, so that line should be changed to `apply_default_cascade(args, constructor=None)`

```suggestion
def apply_default_cascade(args, constructor):
```

Reply:
Great point about consistency with the other functions in the file 👍 Done.